data/human/free worlds 1 start.txt

```diff
@@ -980,6 +980,19 @@ mission "FW Pirates 1"
 			
 			label agree
 			`	"So, here's the plan," he says. "These four systems belong to the pirates." He points to them on the map. "We want you to scout out all four systems. Just a quick fly-through, no need to land on their planets. Then report back to us, and we'll decide what our next step will be."`
+			choice
+				`	"Sounds good."`
+					goto leave
+				`	"What about the pirate systems to the north of Men?"`
+			`	Tomek shrugs. "Simply put, those worlds aren't a threat to us right now. From what we can gather, most of the factions there are focused on internal power struggles at the moment rather than piracy in the Rim. Those CCOR guerrillas are more than enough to keep any pirate warlords there busy for the immediate future."`
+			choice
+				`	"CCOR?"`
+					goto ccor
+				`	"Understood."`
+					goto leave
+			label ccor
+			`	Tomek looks surprised. "I assumed you knew. The Confederated Councils of the Outer Rim was a government that wanted to practice its own particular flavor of communism outside of the Republic's influence. They fell around 50 years ago, and their former planets have since been under the control of a mixture of pirate gangs and CCOR guerrilla groups. I wouldn't worry about the latter too much either - at this point they're little more than a pirate gang themselves."`
```

Comment:
I know your waiting on Ebreus, but at the moment I'm left with questions. Are some planets controlled by pirates and some by ccor guerrillas and are they working together or not? In the first section it's clear the two are in conflict but in the second section it almost sounds like they are working together. I'm not sure what your intent is, but does this make it a little clearer? 
```suggestion
			`	Tomek looks surprised. "I assumed you knew. The Confederated Councils of the Outer Rim was a government that wanted to practice its own particular flavor of communism outside of the Republic's influence. They fell around 50 years ago, and their former planets are now under the control of either pirate gangs or remaining CCOR guerrilla groups. I wouldn't worry about it too much - at this point, the CCOR are little more than a pirate gang themselves."`
```

Reply:
Or maybe even:

```suggestion
			`	Tomek looks surprised. "I assumed you knew. The Confederated Councils of the Outer Rim was a government that wanted to practice its own particular flavor of communism outside of the Republic's influence. They fell around 50 years ago, and their former planets are now under the control of either pirate gangs or guerrilla groups trying to restore the CCOR. I wouldn't worry about the guerrillas too much - at this point, they're little more than a pirate gang themselves."`
```